e2e-test/src/main/resources/wiremock/hxinsight/__files/.secrets.baseline

```diff
@@ -0,0 +1,2 @@
+pragma: allowlist secret
```

Comment:
AFAIK, this file shouldn't be created here. You need to add following definition in `.secrets.baseline` file in the root directory (which you seem to have not included in your branch at all):
```
"e2e-test\\src\\main\\resources\\wiremock\\hxinsight\\__files\\access-token.json": [
      {
        "type": "JSON Web Token",
        "filename": "e2e-test\\src\\main\\resources\\wiremock\\hxinsight\\__files\\access-token.json",
        "hashed_secret": "fe58910a719f8b813574d85fafef95b25e09f309",
        "is_verified": false,
        "line_number": 2,
        "is_secret": false
      }
    ]
```

Reply:
implemented as mentioned above